### ISSUE TITLE
Integration Nexus Vicon into 3D Slicer through PlusServer

### ISSUE THREAD

**Issue by AurelieSar** [PlusToolkit/PlusLib#1208]
Dear PlusToolkit team,

As you may already know, we are working on the integration of Nexus Vicon into 3D Slicer through PlusServer, to perform 3D freehand ultrasound with Vicon and Qualisys systems.

Here are the attached files : 
[Configfile_Server.zip](https://github.com/user-attachments/files/17378836/Configfile_Server.zip)

We already succeeded in : 
- creating a OpenIGTLink server using Python that sends random quaternions into the port 18944 (file "my_server.py" attached in the zip). Let's call him "NexusServer" (later we will send the Vicon Nexus data from the rigidbody into this server)
- creating a OpenIGTLink server using PlusServer that receives data from the NexusServer (18944) and sends data into the port 18946 (config file "my_OpenIGTLinkTracker.xml"), to be read by 3D Slicer. Later, we want to use the PlusServer server to mix data from NexusServer and from the ultrasound, either Telemed or from a videograbber. 
- reading the PlusServer data with 3D Slicer. 

We confirm that, when we launch PlusServer server, we do have a client connection to our NexusServer. We also confirm that 3D Slicer connects to the PlusServer server, by displaying "ON", however we have no transform appearing in "IN". 

We suspect that the data sent into PlusServer from the NexusServer are not in the right format. 

Could you help us with the format of transform needed by PlusServer ? 

Best regards,
Aurélie

**Comment by lassoan** [comment]
You can see the error in the Slicer Python console:

> [VTK] Received unknown device type POSITION, device=Tracker

This means that Slicer ignores `POSITION` messages. You can use `TRANSFORM` messages instead.

**Comment by AurelieSar** [comment]
Yes indeed thank you very much, we realized our error a few hours after. We now have something appearing in "IN", however it is not yet detected as a transform, we must have forgotten a parameter in the TRANSFORM MESSAGE setting in python.

**Comment by lassoan** [comment]
See example for sending transforms here: https://github.com/lassoan/pyigtl/blob/master/examples/example_tracked_image_server.py

**Comment by lassoan** [comment]
Plus always uses the "(FromCoordinateSystem)To(To CoordinateSystem)" as device name. Device name is used as node name in Slicer.
